sql/moz-fx-data-shared-prod/cloudflare/browser_usage/view.sql

```diff
@@ -2,6 +2,23 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.cloudflare.browser_usage`
 AS
 SELECT
-  *
+  brwsr_usg.*,
+  country_codes.location_name,
+  country_codes.region_name
 FROM
-  `moz-fx-data-shared-prod.cloudflare_derived.browser_usage_v1`
+  `moz-fx-data-shared-prod.cloudflare_derived.browser_usage_v1` brwsr_usg
+LEFT JOIN
+  (
+    SELECT
+      'ALL' AS code,
+      'Global' AS location_name,
+      'Global' AS region_name
```

Comment:
Should we add this to the `country_codes_v1` table?

Reply:
I don't know if it's a standard, so I wouldn't feel comfortable adding it - it was more specific to cloudflare, ALL was just the shorthand I used when I didn't pass a specific country in order to get the global level data